templates/workspace_services/azureml/terraform/compute.tf

```diff
@@ -0,0 +1,92 @@
+resource "random_password" "password" {
+  length           = 16
+  lower            = true
+  min_lower        = 1
+  upper            = true
+  min_upper        = 1
+  numeric          = true
+  min_numeric      = 1
+  special          = true
+  min_special      = 1
+  override_special = "_%@"
+}
+
+resource "azurerm_key_vault_secret" "aml_password" {
+  name         = "cp-${local.short_service_id}"
+  value        = random_password.password.result
+  key_vault_id = data.azurerm_key_vault.ws.id
+}
+
+
+resource "azapi_resource" "compute_cluster" {
```

Comment:
What doesn't the native azurerm resource support that made you choose to use the azapi approach? Maybe worth documenting?

Reply:
Added a comment.